django/contrib/auth/admin.py

```diff
@@ -80,9 +82,11 @@ def get_form(self, request, obj=None, **kwargs):
         return super(UserAdmin, self).get_form(request, obj, **defaults)
 
     def get_urls(self):
-        return [
-            url(r'^(.+)/password/$', self.admin_site.admin_view(self.user_change_password), name='auth_user_password_change'),
-        ] + super(UserAdmin, self).get_urls()
+        return [url(
```

Comment:
1. Please try to refrain from making unrelated cleanups like this (when otherwise not touching the code) as it's confusing with looking through commit history.
2. ModelClass instances -> ModelAdmins and Fix -> Fixed in the commit message
3. Thanks!